os/arch/arm/src/armv7-a/arm_assert.c

```diff
@@ -580,7 +580,7 @@ void up_assert(const uint8_t *filename, int lineno)
 
 	irqstate_t flags = irqsave();
 #ifdef CONFIG_SECURITY_LEVEL
-	lldbg_noarg("security level: %d\n", get_security_level());
+	lldbg_noarg("\nsecurity level: %d\n", get_security_level());
```

Comment:
If you modify here, when someone adds logs before this, this should be changed again. And need to be checked whether someone adds `\n` at start. I suggest adding `\n` at start of the function.

Other question is
why only `armv7-a` file is changed? how about `armv8-m`, `armv7-m`, `armv7-r`, hardfault? memfault and other fault files?

Reply:
Your suggest means that adding `\n` at start of up_assert?

As I mentioned in commit message, I only added in `armv7-a` because log continues only in `armv7-a`. 
`arm7-m, arm7-r, arm8-m` is not occurred.